questionpy_sdk/webserver/app.py

```diff
@@ -57,18 +57,49 @@ async def render_options(request: web.Request) -> web.Response:
 
 @routes.post('/submit')
 async def submit_form(request: web.Request) -> web.Response:
-    """Get the options form definition and the form data on."""
+    """Store the form_data from the Options Form in the StateStorage."""
     webserver: 'WebServer' = request.app['sdk_webserver_app']
     form_data = await request.json()
+
+    async with webserver.worker_pool.get_worker(webserver.package, 0, None) as worker:
+        form_definition, _ = await worker.get_options_form(None)
```

Comment:
Hier (und in `/repeat`) sollten wir schon den bisherigen State mitschicken, wenn es einen gibt.